README.md

```diff
@@ -1,29 +1,52 @@
-# BETA Engine
+# BETA (β) Engine
 #### A choose-your-own-adventure game engine!
 
 ## What is this?
 A DSL that allows a user to create their own choose-your-own-adventure game. Created for UBC CPSC 410.
 
+Built on Java 8 with JDK version 1.8.0_181
+
+## Try it out!
+Download the .jar [here](https://drive.google.com/open?id=1qwTiAtOuj0PNbt0xrk85thoUz5RO7cv1)!
+
 ## Game Features
-* Lives: allow for the user to make X incorrect choices and redo the last scene
-* Timed choices: constrain the user to making their choices in X seconds or it's game over
-* Choices: options for your player to progress the game
-* Exposition choices: choices that don't progress the story. Perfect for creating cutscenes
-* Conditional scenes: scenes that appear based on choices you made earlier in the game
-* Conditional choices: choices that appear based on choices you made earlier in the game
 * Start screens: display the title of your game in style
 * Multiple end screens: construct different ending for different player choices
+* Choices: options for your player to progress the game
+* Exposition choices: choices that don't progress the story. Perfect for creating cut scenes
+* Conditional choices: choices that appear based on choices you made earlier in the game
+* Timed choices: constrain the user to making their choices in X seconds or it's game over
 
 ## Technical Features
-* Constrained image positioning: position your image on screen left, right, top, bottom, center
+* Constrained image positioning: Layer and position your image on screen left, right, top, bottom, center
 * BGM and sound effects: shape the atmosphere with unique background music and sound effects for every screen
 
-## Code Example
+## Grammar
+```
+PROGRAM 		::= GAME_SETTINGS START_SCENE STORY_SCENES DEATH_SCENES END_SCENES
+GAME_SETTINGS	::= "GAME:" "title:" STRING "description:" STRING STATS
+STATS			::= "stats" STAT*
+STAT			::= NAME [INTEGER|STRING]
+START_SCENE     ::= "START SCENE:" NAME CHOICE* [TEXT|BGM|SOUND_EFFECT|PICTURE]*
```

Comment:
[TEXT|BGM...]* can use ? to mean at most 1. But we don't handle that in the code anyways, so idk if we should change the grammar for that

Same with the other [..|..]* ones